src/ipkm.f

```diff
@@ -3,15 +3,19 @@
 C>
 C> @author J. Woollen @date 1994-01-06
 
+C> Encode an integer value within a character string.
+C>
 C> This subroutine encodes an integer value within a specified
 C> number of bytes of a character string, up to a maximum of 8
 C> bytes.
 C>
-C> @param[in] N      -- integer: Value to be encoded
-C> @param[in] NBYT   -- integer: Number of bytes of CBAY (up to a
-C>                      maximum of 8) within which to encode N
-C> @param[out] CBAY  -- character*(*): String of length NBYT bytes
-C>                      containing encoded integer N
+C> This subroutine is the logical inverse of subroutine iupm().
+C>
+C> @param[in] N - integer: Value to be encoded.
+C> @param[in] NBYT - integer: Number of bytes of CBAY (up to a
+C> maximum of 8) within which to encode N.
+C> @param[out] CBAY- character*(*): String of length NBYT bytes
```

Comment:
Not sure if the lack of space before the hyphen will mess up the doxygen parsing

Reply:
Good catch.  I'll make sure I get that fixed in my next PR.